projects/power_distribution/test/test_pd_fan_ctrl.c

```diff
@@ -201,15 +202,21 @@ void test_front_pd_fan_ctrl_pot(void) {
   // Check max, min, and 50% potentiometer values transmit correct
   // values and execute properly
 
-  // set adc to return max value
+  // set adc to open value
   adc_ret_val = ADC_MAX_VAL;
   TEST_ASSERT_EQUAL(STATUS_CODE_OK, pd_fan_ctrl_init(&s_fan_settings, true));
   delay_ms(REAR_FAN_CONTROL_REFRESH_PERIOD_MILLIS + 10);
+  TEST_ASSERT_EQUAL(FAN_MIN_I2C_WRITE, i2c_buf1[1]);
+  TEST_ASSERT_EQUAL(FAN_MIN_I2C_WRITE, i2c_buf2[1]);
+
+  // set adc to return max value
+  adc_ret_val = FRONT_FAN_CTRL_MAX_VALUE_MV;
+  delay_ms(REAR_FAN_CONTROL_REFRESH_PERIOD_MILLIS + 10);
```

Comment:
I noticed that line 219 and 208 have a delay with a 10 ms tolerance. Why is this not needed in line 220?

Reply:
Hm not sure. That's a @mitchellostler question. I simply used the same format again to test for 3.3V

Reply:
I'm pretty sure the tolerance was just added to ensure that values are available. If the tests are passing it doesn't really need a change, but might be a good idea to add it to keep it consistent

Reply:
Sure I can do that